k8s/operator.go

```diff
@@ -0,0 +1,127 @@
+package k8s
+
+import (
+	"context"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	tykModePro = "pro"
+	tykModeKey = "TYK_MODE"
+	tykAuthKey = "TYK_AUTH"
+	tykOrgKey  = "TYK_ORG"
+	tykURLKey  = "TYK_URL"
+)
+
+// BootstrapTykOperatorSecret bootstrap a Kubernetes Secret utilized by Tyk Operator.
+// If the system has the secret created already, it deletes the existing one and recreates
+// a secret for Tyk Operator.
+func (c *Client) BootstrapTykOperatorSecret() error {
+	secrets, err := c.clientSet.
+		CoreV1().
+		Secrets(c.appArgs.K8s.ReleaseNamespace).
+		List(context.TODO(), metav1.ListOptions{})
```

Comment:
Why don't we use Get() here since we have secret name?

Reply:
I was also thinkinig the same thing but didn't want to update code.
But i'll update the code piece as you suggested

Reply:
I used Delete directly